finish/pom.xml

```diff
@@ -77,6 +77,18 @@
         <!-- end::profile-docker-image[] -->
     </profiles>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.openliberty.features</groupId>
+                <artifactId>features-bom</artifactId>
+                <version>18.0.0.2</version>
```

Comment:
Why is this version hardcoded to 18002?

Reply:
It is required to specify a version, otherwise you see this error when you try to build it:

![image](https://user-images.githubusercontent.com/8409464/44402270-4087a200-a51f-11e8-9833-8067a4c7849e.png)